### ISSUE TITLE
About the backbone's pretrained model

### ISSUE THREAD

**Issue by merisgong208** [zylo117/Yet-Another-EfficientDet-Pytorch#375]
I trained my custom data according to the tutorial, with pretrained xx_d0.pth, but I the result is terrible. Seems cannot converge (although the loss is smll, e.g. <0.1).

My dataset has no problem, for I have trained on other models.

As to this project, I think the main problem is on the backbone.
Having not tried, but I suppose with a pretrained efficientnet model, the result should be normal.

Want to talk about this with guys.

**Comment by fengz63** [comment]
> I trained my custom data according to the tutorial, with pre-trained xx_d0.pth, but the result is terrible. It seems cannot converge (although the loss is small, e.g. <0.1).
> 
> My dataset has no problem, for I have trained on other models.
> 
> As to this project, I think the main problem is on the backbone.
> Having not tried, but I suppose with a pre-trained efficientnet model, the result should be normal.
> 
> Want to talk about this with guys.

There are links of pretrained efficientdet models in README
